### ISSUE TITLE
[BUG] Generating text is timing out 

### ISSUE THREAD

**Issue by sejalgupta** [mlfoundations/open_flamingo#263]
I am trying to run the Generating Text from the README.md. However, every time I run the command below it never finishes running and I have to manually stop it. I have waited like 5-10 minutes each time. 

```
generated_text = model.generate(
        vision_x=vision_x,
        lang_x=lang_x["input_ids"],
        attention_mask=lang_x["attention_mask"],
        max_new_tokens=20,
        num_beams=3,
    )
```

**Comment by anas-awadalla** [comment]
Please follow the format for [bug reports](https://github.com/mlfoundations/open_flamingo/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+YOUR+DESCRIPTION+HERE) so we can reproduce the issue. For instance it would be very useful to know which model you are running and where you are running it (are you using a GPU etc.).
